app/assets/stylesheets/modules/forms.scss

```diff
@@ -83,6 +83,12 @@ hr {
   display: none;
 }
 
+// This is done because the html5 shim removes elements
+// with the .help-block class after selecting a date
```

Comment:
I'll update the comment to be more clear.

I don't think we can reliably test this because the HTML5 shim + Capybara/Poltergeist don't play as nicely as they could.